test_collections/sdk_tests/support/python_testing/models/python_test_parser.py

```diff
@@ -129,6 +139,14 @@ def __test_case_names(methods: list[FunctionDefType]) -> list[str]:
             if match := re.match(TC_TEST_FUNCTION_PATTERN, m.name):
                 if name := match["title"]:
                     test_names.append(name)
+            # THIS IS A WORKAROUND CODE for TE2
```

Comment:
Do we need to create an issue to track the workaround remove?

Reply:
And maybe also link the issue created in the comments would be good for future reference.

Reply:
Done.
Original issue: # Issue: https://github.com/project-chip/certification-tool/issues/152
Issue to remove the workaround: https://github.com/project-chip/certification-tool-backend/issues/62